assets/js/components/SaptuneDetails/SaptuneDetails.test.jsx

```diff
@@ -22,7 +22,7 @@ describe('SaptuneDetails', () => {
     };
 
     const customSapconfService = {
-      active: 'inactive',
+      active: 'active',
```

Comment:
why changes in this file?

Reply:
it was a leftover from the previous test, so more  varied icons can be rendered.
I will revert it